tests/netdev_test/main.c

```diff
@@ -198,11 +198,11 @@ static int test_receive(void)
         puts("=================");
         puts("expected");
         puts("=================");
-        puts(gnrc_netif_addr_to_str(_test_src, ETHERNET_ADDR_LEN, addr_str));
+        puts(netif_addr_to_str(_test_src, ETHERNET_ADDR_LEN, addr_str));
         puts("=================");
         puts("received source");
         puts("=================");
-        puts(gnrc_netif_addr_to_str(gnrc_netif_hdr_get_src_addr(hdr->data),
+        puts(netif_addr_to_str(gnrc_netif_hdr_get_src_addr(hdr->data),
```

Comment:
I said keep the old tests... I would prefer a dedicated test application using `embUnit` for this.